Makefile

```diff
@@ -26,6 +28,18 @@ else
 	composer update --prefer-dist
 endif
 
+handlebars:
+ifeq (,$(handlebars))
+	@echo "No handlebars command available, remember to install this"
+	@echo " # npm install handlebars -g"
+	@echo "Then you must compile the templates"
```

Comment:
I would not go into explaining to compile templates. Imagine users do first time `make` and this blows up and he thinks he should execute `make js-templates`:) Consider removing it or just telling "Then run make again" or "Then restart your command"...

Reply:
Well, Compiling the templates, must be a rule just as compile stylesheets from Sass/less.
Maybe we must add a 'build' rule, that download the dependencies, and build the templates, but these must be the rules separately.